LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
Why was this changed to MIT?

Reply:
Following https://github.com/google/yapf/blob/master/LICENSE in another google open source project

Reply:
I ended up switching back when I followed your [comment](https://github.com/jackdavidweber/cjs_capstone/pull/125#issuecomment-667326421) on how to generate license